README.MD

```diff
@@ -124,14 +151,18 @@ McpServerOptions options = new()
                         {
                             Name = "echo",
                             Description = "Echoes the input back to the client.",
-                            InputSchema = new JsonSchema()
-                            {
-                                Type = "object",
-                                Properties = new Dictionary<string, JsonSchemaProperty>()
+                            InputSchema = """
```

Comment:
Given that there's no `JsonSerializer.Deserialize` I take it you've replaced `JsonElement` with string?

Reply:
No, that's a typo, this needs a deserialize.